src/Raven.Client/Documents/Session/Operations/BatchOperation.cs

```diff
@@ -328,7 +329,18 @@ private void HandlePatch(BlittableJsonReaderObject batchResult)
                     }
 
                     if (documentInfo.Entity != null)
-                        _session.EntityToBlittable.PopulateEntity(documentInfo.Entity, id, documentInfo.Document, _session.JsonSerializer);
+                    {
+                        var old = _session.JsonSerializer.ObjectCreationHandling;
+                        _session.JsonSerializer.ObjectCreationHandling = ObjectCreationHandling.Replace;
```

Comment:
What should happen if I have a patch like
`this.Address.City = 'abc'`
? 

The `Address` will be a new instance or old? What behavior do we want? Only the root entity instance cannot be changed?

Reply:
It will create a new instance, yes.
The root instance is the only one we care about here, yes.